File: rs/src/models/public_key.rs
use serde::{Deserialize, Serialize};

#[derive(Debug, Clone, Serialize, Deserialize, sqlx::FromRow)]
pub struct PublicKey {
    #[sqlx(rename = "Id")]
    pub id: String,
    #[sqlx(rename = "SessionId")]
    pub session_id: String,
    #[sqlx(rename = "Alias")]
    pub alias: String,
    #[sqlx(rename = "PublicKeyJson")]
    pub public_key_json: String,
    #[sqlx(rename = "DateCreated")]
    pub date_created: String,
}

File: src/JustSending/Data/Models/PublicKey.cs
using System;

namespace JustSending.Data.Models
{
    public class PublicKey
    {
        public string Id { get; set; }
        public string SessionId { get; set; }
        public string Alias { get; set; }
        public string PublicKeyJson { get; set; }
        public DateTime DateCreated { get; set; }
    }
}
